ramp-database/ramp_database/tools/submission.py

```diff
@@ -626,10 +644,8 @@ def set_predictions(session, submission_id, path_predictions):
     path_predictions : str
         The path where the results files are located.
     """
-    all_cv_folds = (session.query(SubmissionOnCVFold)
-                           .filter_by(submission_id=submission_id)
-                           .all())
-    for fold_id, cv_fold in enumerate(all_cv_folds):
+    sub = get_submission_by_id(session, submission_id)
```

Comment:
As a side note I'm wondering about the need for some these helper functions. For instance, `get_submission_by_id` will call internally `select_submission_by_id` and then call 
```py
    submission.event.name
    submission.team.name
```
Unless I'm mistaken that's 3 DB queries, for something that could have been just,
```py
sub = (session.query(Submission)
                   .filter(Submission.id == submission_id)
                   .first())
```
it's slightly more verbose, but much more explicit about what is going on..


And actually I find the previous code more explicit without with fewer sqlalchemy magic. Are you saying this is faster or am I missing something?

Reply:
I agree. I think that we add some refactoring to create some functions for some query but I am unsure of the use indeed. The thing is that, you finish up by writing the query instead of checking that the query was already written. I would be inclined to remove these queries and just be explicit.